debian-rootfs.yml

```diff
@@ -0,0 +1,59 @@
+---
+- hosts: localhost
+  connection: local
+  roles:
+    - common
+    - debootstrap
+    - role: linux-kernel
+      version: "{{ linux_4_9 }}"
+      config: "{{ apu_config }}"
+    - role: linux-kernel
+      version: "{{ linux_4_14 }}"
+      config: "{{ apu_config }}"
+    - chroot_mount
+
+#TODO: make variables visible in all roles
```

Comment:
I think if we want to have variables to be visible in all roles (i.e. whole playbook scope), we should not define them in any of the specific roles, but use `vars_files` for example: https://docs.ansible.com/ansible/2.6/user_guide/playbooks_variables.html#variable-file-separation